### PR TITLE
docs: remove application docs w/signoff

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**

This PR fixes #
Remove the Meshery Application docs.
https://layer5io.slack.com/archives/C04LJS10VTL/p1716390187759729

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit fb8afaa2439f1402f52faea84bfcf75ea0c93742 at: https://664e0cff2acb80009e0c5c04--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit d22eea794a9148eefe354b99f2fcbecb875b016c at: https://664e0ed11810ff03677ef839--meshery-docs-preview.netlify.app

File: docs/pages/guides/configuration-management/configuration-management.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Configuring your Cloud Native Infrastructure and Applications
```
Comment (MUzairS15) [inline docs/pages/guides/configuration-management/configuration-management.md:3]: We should keep it here imo, as a user they would refer to apps as what we mean by designs.
But only remove mention of applications where it's mentioned/described as a meshery construct.

File: docs/pages/guides/configuration-management/configuration-management.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ redirect_from:
   - /tasks/patterns.md
 ---
 
-As an cloud native management platform, Meshery has built-in support infastructure and application configuration management. Use Meshery to configure your multi-cluster Kubernetes environments on-premises or across clouds.
+As an cloud native management platform, Meshery has built-in support infastructure configuration management. Use Meshery to configure your multi-cluster Kubernetes environments on-premises or across clouds.
```
Comment (MUzairS15) [inline docs/pages/guides/configuration-management/configuration-management.md:15]: Same here as well.

**Review by MUzairS15 — DISMISSED** [review]
Else, LGTM

**Comment by l5io** [comment]
🚀 Preview for commit 0fff413ef49b85f509f5cf930db15f13ae71b083 at: https://664e2ad7bac6f500a41dcec9--meshery-docs-preview.netlify.app

**Review by MUzairS15 — APPROVED** [review]

**Pull request merged**